static/scss/results.scss

```diff
@@ -1,3 +1,58 @@
+.results-nav {
+  margin-top: 20px;
+}
+.sticky {
+  background-color: white;
+  padding-bottom: 20px;
+  border-bottom: 1px solid #f6f6f6;
+  z-index: 9999;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  @media (max-width: 600px) {
+    padding-bottom: 10px;
+  }
+}
+:focus {
+  outline: none;
+  border: none;
+}
```

Comment:
Just noticed the focus change. This is pretty broad reaching and will make things hard for users who use a keyboard for screen navigation because they can't use a mouse. What's the idea for adding it?

Reply:
long story short, a bad/hacky way to make the styles act the same (for whatever reason it's not adding focus styles to key races) -- meant to go back and edit but forgot. they're gone now

Reply:
Gotcha. Should be good to merge once tests pass.